lib/dom_dsl_core.ml

```diff
@@ -32,7 +32,9 @@ module Common = struct
   end
 
   let fragment children = Core.Fragment.make ~children ()
+  let none = fragment []
```

Comment:
Why not
```suggestion
  let none = Core.null
```
?

As you mentioned in your comment.

Reply:
Just because I didn't remember it existed when I wrote this, and now that I do I wonder if it should be called `none` or `null` in the DSL. I'm leaning towards `none` though.